pkg/cable/driver.go

```diff
@@ -20,3 +20,9 @@ type Driver interface {
 	// DisconnectFromEndpoint disconnects from the connection to the given endpoint.
 	DisconnectFromEndpoint(endpoint types.SubmarinerEndpoint) error
 }
+
+// Function prototype to create a new driver
+type DriverCreateFunc func(localSubnets []string, localEndpoint types.SubmarinerEndpoint) (Driver, error)
+
+// Static map of supported drivers
+var Drivers = map[string]DriverCreateFunc{}
```

Comment:
I think we should go thru a function to add drivers to prevent dups: 

```
var drivers = map[string]DriverCreateFunc{}

func AddDriver(name string, driverCreate DriverCreateFunc) {
	if drivers[name] != nil {
		panic(fmt.Sprintf("Multiple cable engine drivers attempting to register with name %q", name))
	}
	
	drivers[name] = driverCreate
}
```

Reply:
done